src/libraries/Directory.Build.targets

```diff
@@ -1,5 +1,8 @@
 <Project InitialTargets="AddSkipGetTargetFrameworkToProjectReferences">
   <PropertyGroup>
+    <TargetsNetFx Condition="$(TargetFramework.StartsWith('net4'))">true</TargetsNetFx>
```

Comment:
Should we just remove these entirely? Are these used anywhere in targets other than in this file?

Reply:
these are used for itemgroups. we just removed their occurrences from propertyGroups. 
The reason behind this was to keep the change in developer experience to be minimum

Reply:
I think we should get rid of these helper properties as part of this PR. They will just confuse devs as as they will most likely try to use them on properties.

Reply:
Right, that is why I was asking that. Now devs will need to remember that this can only be used on itemgroups.

However, should we do that removal as part of the infra batched rollouts? That is a breaking change in the infra world for libraries.

Reply:
> However, should we do that removal as part of the infra batched rollouts? That is a breaking change in the infra world for libraries.

That change would then need to wait until June, 1st as we already sent out the list of planned changes. I'm not 100% certain that this counts as a breaking change.

Reply:
> I'm not 100% certain that this counts as a breaking change.

You could argue that making these Targets* helper properties useless is already a breaking change.

Reply:
If you think that after removing those, you go as a developer and try to use them and then your Compile items are messed up because those properties are no longer defined and you have to investigate and know that you need to use `TargetFramework.StartsWith`, to me sounds like a breaking change in that scenario 😢

Reply:
>You could argue that making these Targets* helper properties useless on properties is already a breaking change.

Yeah I agree.

Reply:
> If you think that after removing those, you go as a developer and try to use them and then your Compile items are messed up because those properties are no longer defined and you have to investigate and know that you need to use TargetFramework.StartsWith, to me sounds like a breaking change in that scenario 😢

Agreed. OK, I'm fine with sending a follow-up mail with the additional change. I don't see another way to not block us from making progress here.

Reply:
OK spoke with @Anipik offline and he will do the removal of `TargetsNetCoreApp`, `TargetsNetFx` and `TargetsNetStandard` in a follow-up PR which is part of the batched rollout for May: https://github.com/dotnet/runtime/issues/35202. I will send out a notice via mail now.

Reply:
I can remove this stuff entirely in a follow up pr